bases/shared/entrypoint.sh

```diff
@@ -353,7 +355,12 @@ start_chain () {
         fi
         (cd /usr/src/agoric-sdk && node $extra /usr/local/bin/ag-chain-cosmos --home "$AGORIC_HOME" start --log_format=json $@  >> /state/app.log 2>&1)
     else
-        $(ag_binary) start --home="$AGORIC_HOME" --log_format=json $@ >> /state/app.log 2>&1 
+        $(ag_binary) start --home="$AGORIC_HOME" --log_format=json $@ >> /state/app.log 2>&1
+
+        if [ "$AUTO_APPROVE_PROPOSALS" = "true" ]; then
+            ./scripts/auto-approve.sh $(ag_binary) &
```

Comment:
This script may need to be mounted into the pod. 
Example: https://github.com/Agoric/instagoric/blob/37ca8b8b8d86fa4521b33736214dd88f255cb77e/bases/node/node.yaml#L328

Reply:
Yes. Thanks for reminding. Will do while testing on k8 and then push the change.